### ISSUE TITLE
Add Missing Localiztion

### ISSUE THREAD

**Issue by righthandofvecna** [pokemon-tabletop-reunited/ptr2e#595]
Missing localization for:

- `PTR2E.Dialog.DeleteAction.Content`
- `PTR2E.Dialog.DeleteAction.Title`
